tests/Avalonia.Controls.UnitTests/MenuItemTests.cs

```diff
@@ -56,7 +62,7 @@ public void MenuItem_Is_Disabled_When_Bound_Command_Doesnt_Exist()
         }
 
         [Fact]
-        public void MenuItem_Is_Disabled_When_Bound_Command_Is_Removed()
+        public void MenuItem_Is_Disabled_Bound_Command_Is_Removed()
```

Comment:
Why the change in name here? The removal of "when" makes it grammatically incorrect.